packages/core/bootstrap/src/lib/middleware/burst-limit/index.ts

```diff
@@ -22,14 +22,21 @@ const availableSecondLimitCapacity = async (
   store: Store<BurstLimitState>,
   burstCapacity1s: number,
 ) => {
+  const enabled = parseBool(getEnv('RATE_LIMIT_CAPACITY'))
```

Comment:
We can't parse this as bool. We should instead be checking if it's undefined.

Also suggest renaming this constant to something more descriptive (`customRateLimitCapacity`?)